amazon-msk/cdk/lib/IotIngestAndControlStack.ts

```diff
@@ -415,43 +413,46 @@ export class IotIngestAndControlStack extends cdk.Stack {
 
     autoScalingGroup.attachToNetworkTargetGroup(targetGroup);
 
-    const metricsNamespace = context.cloudwatch?.metrics?.namespace ?? 'zilla';
+    if (context.cloudwatch?.metrics) {
+      const metricsNamespace = context.cloudwatch.metrics.namespace;
 
-    const metricWorkerUtilization = new cw.Metric({
-      namespace: metricsNamespace,
-      metricName: 'engine.worker.utilization',
-      statistic: 'Average',
-      period: cdk.Duration.minutes(5),
-    });
+      const metricWorkerUtilization = new cw.Metric({
+        namespace: metricsNamespace,
+        metricName: 'engine.worker.utilization',
+        statistic: 'Average',
+        period: cdk.Duration.minutes(5),
```

Comment:
I was thinking this should probably be configurable, does it make sense to align with the `interval` at which we publish the metrics?